frontend/src/pages/Admin/Components/Testimonial/AddTestimonial/AddTestimonial.jsx

```diff
@@ -11,7 +11,7 @@ export function AddTestimonial() {
     name: "",
     position: "",
     company: "",
-    image: "https://i.pinimg.com/originals/f4/cd/d8/f4cdd85c50e44aa59a303fb163ff90f8.jpg",
+    image: "",
```

Comment:
There should be a default image incase admin doesn’t provide it ‘coz this will appear on home page

Reply:
@Kajol-Kumari  yes it is implemented.